src/support_sphere_py/src/support_sphere/models/public/cluster.py

```diff
@@ -41,3 +45,4 @@ class Cluster(BasePublicSchemaModel, table=True):
     geom: Geometry|None = Field(sa_type=Geometry(geometry_type="POLYGON"), nullable=True)
 
     households: list["Household"] = Relationship(back_populates="cluster", cascade_delete=False)
+    user_captain_clusters: list["UserCaptainCluster"] = Relationship(back_populates="cluster", cascade_delete=False)
```

Comment:
```suggestion
    user_captain: list["UserCaptainCluster"] = Relationship(back_populates="cluster", cascade_delete=False)
```
can we rename this to `user_captain` instead of the actual table name?